crates/tests/standalone/build.rs

```diff
@@ -44,6 +44,7 @@ fn main() {
         &[
             "Windows.Win32.Networking.WinSock.WSADATA",
             "Windows.Win32.UI.WindowsAndMessaging.GetWindowLongPtrW",
+            "Windows.Win32.System.Diagnostics.Debug.RtlCaptureContext",
```

Comment:
Let put this in a new test module, say `b_arch_dependency`. That way we can easily validate that the change doesn't perturb anything else.